test/Reflection/ReflectionClassTest.php

```diff
@@ -18,6 +18,12 @@ private function getComposerLocator()
         return new ComposerSourceLocator($loader);
     }
 
+    /**
+     * @covers \BetterReflection\Reflection\ReflectionClass::inNamespace()
```

Comment:
Why? Surely doing it this way means you specify the exact method so you
don't even have in-class coverage leaking. Each test is therefore
restricted to only covering the method we intended to test.